Lib/test/test_pdb.py

```diff
@@ -2771,6 +2770,21 @@ def test_package_without_a_main(self):
             "'t_pkg.t_main' is a package and cannot be directly executed",
             stdout)
 
+    def test_nonexistent_module(self):
```

Comment:
Should we assert here that ``os_helper.TESTFN`` doesn't exist (in case some other test forgot to delete it)? It might help someone debug it someday if that happens.